FE/src/stores/filterStateReducer.js

```diff
@@ -0,0 +1,28 @@
+import { useCallback } from 'react';
+
+export const initialFilterState = {
+  isOpen: true,
+  author: null, // 작성자
+  labels: null, // 레이블
+  milestone: null, // 마일스톤
+  assignees: null, // 담당자
+  comments: null // 댓글
+};
+
+export const filterStateReducer = (state, action) => {
+  const { type, payload } = action;
+
+  switch (type) {
+    case 'RESET-FILTER': {
+      return initialFilterState;
+    }
+    case 'FILTER-ISSUES': {
+      return {
+        ...initialFilterState,
+        ...payload
+      };
+    }
```

Comment:
현재 상태를 기반으로 상태 변경이 일어나야 해요!
```suggestion
    case 'FILTER-ISSUES': {
      return {
        ... state,
        ...payload
      };
    }
```